### PR TITLE
Use a for loop not some

### PR DESCRIPTION
**Pull request by Teapane**
### What does this pull request do?
*Briefly summarize any features implemented, bugs fixed, or other code changes.*

- `IE11` only supports `some()` on dynamically generated arrays...
🤔 🤔 🤔 

#### Required before merge:
- [ ] Has tests (where applicable)
- [ ] Has appropriate documentation (comments, readme, etc.)
- [ ] Passes CircleCI & Heroku builds

### What is the relevant story?
*Link to the story if there is one. If not, briefly explain why this change was needed.*

[[Title of the story]]( https://app.clubhouse.io/guild/story/ )

### How do you feel?
*Attach a GIF that describes how you feel about what you wrote or implemented. Feel free to include additional context!*

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -32,7 +32,13 @@ function notABot() {
...
     'ia_archiver',
   ];
-  return !botList.some(botString => user.includes(botString));
+  for(x in botlist) {
+    if(user.includes(x)) {
+      return true
+    } else {
+      return false
+    }
+  }
```
Comment (aprildagonese) [inline index.ts]: If it is a bot, it should return false.

File: botUtilities.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    'facebookexternalhit',
+    'facebot',
+    'ia_archiver',
+  ];
+  return findUserInBotList(botList, user);
+}
+
+function findUserInBotList(botList: string[], user: string) {
+  for (let x of botList) {
+    if (user.includes(x)) {
```
Comment (elanalynn) [inline botUtilities.ts]: IE does not like `includes`. Can we do this another way?

File: botUtilities.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    'facebookexternalhit',
+    'facebot',
+    'ia_archiver',
+  ];
+  return findUserInBotList(botList, user);
+}
+
+function findUserInBotList(botList: string[], user: string) {
+  for (let x of botList) {
+    if (user.indexOf(x) !== -1 || x.indexOf(user) !== -1) {
```
Comment (elanalynn) [inline botUtilities.ts]: I replaced `includes` with `indexOf` here.

Comment (Teapane) [reply botUtilities.ts]: vintage

**Pull request merged**